### ISSUE TITLE
Import GPX generated by runnerup

### ISSUE THREAD

**Issue by Leonardo-Boss** [jonasoreland/runnerup#1200]
Hi, I was importing an old db and ended up losing some runs in the process. Luckily I still have the gpx and tcx files exported by runnerup of the runs I lost saved. I understand currently there is no way to import these files back into runnerup, and that it's not one objective of the project to support importing these files from other tracker apps as stated in https://github.com/jonasoreland/runnerup/issues/1151. But what about importing the gpx files generated by runnerup in that case there wouldn't be a problem with there not being a standard.

If there's no interest in that I could try writing something, although I don't have experience in android development.

**Comment by gerhardol** [comment]
Of course, if you see a need to import gpx/tcx and submit a PR with reasonable code it will be accepted even I personally do not prioritize this feature.
